app/Models/Port.php

```diff
@@ -269,6 +278,11 @@ public function fdbEntries()
         return $this->hasMany(\App\Models\PortsFdb::class, 'port_id', 'port_id');
     }
 
+    public function groups()
+    {
+        return $this->belongsToMany(\App\Models\PortGroup::class, 'port_group_port', 'port_id', 'port_group_id');
```

Comment:
you shouldn't even need to add the extra arguments there

Reply:
don't know what you exactly mean, i specified the relevant columns and class